### PR TITLE
RadioButtonGroup Examples and Guidance

### PR DESCRIPTION
**Pull request by halocline**
[Preview](https://deploy-preview-735--keen-mayer-a86c8b.netlify.app/components/radiobuttongroup)

- [x] Figma
- [x] Guidance
- [x] Examples
- [x] Theme -- [PR here](https://github.com/grommet/grommet-theme-hpe/pull/75)
- [ ] Grommet -- Will have several issues created to represent outstanding work

Updates to `aries.js` to be removed once https://github.com/grommet/grommet-theme-hpe/pull/75 is merged

### CODE REVIEW

**Comment by vavalos5** [comment]
First pass on RadioButton Group:

### **Radio Button Group**

A Radio Button Group is a set of related options within an element that allows only one entry selected at a time.

### **Guidance**

Keep labels concise and simple
Avoid long text descriptions on labels
If there are more than 5 options, or if the default value is reasonable most of the time, consider using Select instead, to avoid cluttering the experience 
Only use when a user needs to see all related options
Radio Buttons should always be listed vertically with one choice per line
Never ad sub-categories to a Radio Button Group


### **When to use Radio Button Group:**

Radio Button Group is used to lay out a small list of mutually related options that are easily visible. Radio Button Groups always require one choice, so it is important to be clear of what you ask from the user on the label.

If you would like the user to check more than one option, use Checkbox Group instead.

### **Variants**
Radio Button Groups are primarily used to select one single item from a list of two or more options. Here are some ways to use them:

**Validation**
Validation styling indicates that a selection was not made. An error message is accompanied by the styling that explains how to resolve the problem.

**Disabled**
Indicated that the Radio Button Group cannot be interacted with.

**Comment by halocline** [comment]
Great. @vavalos5, I will drop into the page for folks to review.

File: aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  const onChange = value => {
+    setPowerRegulation(value);
+    if (typeof powerRegulation === 'undefined') {
+      setMessage();
+    }
+  };
+
+  return (
+    <Form>
```
Comment (taysea) [inline aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js:27]: When I click on one of these radio buttons, the page goes to an error screen.

File: aries-site/src/pages/components/radiobuttongroup.js [hunk 2]
```diff
@@ -21,13 +35,106 @@ const RadioButtonGroup = () => {
...
             docs="https://v2.grommet.io/radiobuttongroup?theme=hpe#props"
+            width="medium"
           >
             <RadioButtonGroupExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>Keep labels concise and simple.</SubsectionText>
```
Comment (taysea) [inline aries-site/src/pages/components/radiobuttongroup.js:50]: This feels a little awkward/orphaned to have separated from the other best practices.

Comment (halocline) [reply aries-site/src/pages/components/radiobuttongroup.js:50]: Good call. Will move into lead of best practices.

File: aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js (hunk 1 above)
Comment (halocline) [reply aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js:27]: will fix. Thanks.

Comment (halocline) [reply aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupValidationExample.js:27]: Fixed.

File: aries-site/src/pages/components/radiobuttongroup.js (hunk 2 above)
Comment (halocline) [reply aries-site/src/pages/components/radiobuttongroup.js]: implemented

**Pull request merged**